packages/cli/webpack.config.js

```diff
@@ -49,7 +50,25 @@ function getCommonConfig(version, generateTypings) {
                 __IMAGE_VERSION__: JSON.stringify(version),
             }),
             new copyFilesPlugin([{ from: './../crawler/dist/browser-imports.js', to: '.' }]),
-        ].concat(generateTypings ? [] : new ForkTsCheckerWebpackPlugin()), // only add if transpileOnly is true
+        ]
+            .concat(
+                generateTypings
+                    ? {
+                          apply: (compiler) => {
+                              compiler.hooks.afterEmit.tap('BundleDtsFiles', (_) => {
+                                  exec(
+                                      'dts-bundle-generator --project tsconfig.sdk.json src/index.ts -o dist/index.d.ts',
```

Comment:
could require/call js function instead of exec here, but would need to copy over writing-to-file and logs https://github.com/timocov/dts-bundle-generator/blob/9f09015dc3b804adfb64e5ea4654327e3d7262b4/src/bin/dts-bundle-generator.ts#L222